android/kotlin-central-login-oidc/README.md

```diff
@@ -0,0 +1,23 @@
+<p align="center">
+  <a href="https://github.com/ForgeRock/sdk-sample-apps">
+    <img src="https://cdn.forgerock.com/logo/interim/Logo-PingIdentity-ForgeRock-Hor-FullColor.svg" alt="Logo">
+  </a>
+  <hr/>
+</p>
+
+# OIDC app using Kotlin for PingAM, PingOne Advanced Identity Cloud, PingFederate, PingOne for Customers
+
+Ping provides these Android samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
+
+### Integrate with PingAM, PingOne Advanced Identity Cloud, PingFederate, PingOne for Customers:
+
+### Update the PingConfig Class in config.kt
+
+- Modify the PingConfig class to include or remove the necessary fields based on your environment's requirements.
+- Ensure the PingConfig.toFROptions() method is updated to map the appropriate values correctly.
+
+## Requirements
+
+- Latest version of Android Studio
+- Ping Advanced Identity Cloud or Ping AM 7.1 and above or PingFederate or PingOne for Customers
+- Android API level 23+
```

Comment:
I think would be nice to developers to have a "Get started" section in the README of all samples apps. Like what was done in the `java-authenticator` project.